server/src/lib/model/db.ts

```diff
@@ -20,14 +23,14 @@ import {
   findVariantsBySentenceIdsInDb,
 } from '../../application/repository/variant-repository'
 const MINUTE = 1000 * 60
-const DAY = MINUTE * 60 * 24
+const HOUR = MINUTE * 60
+const DAY = HOUR * 24
+const THREE_WEEKS = DAY * 3 * 7
```

Comment:
Then let's be consistent and do:
```suggestion
const WEEK = DAY * 7
const THREE_WEEKS = WEEK * 3
```
;)